### DISCUSSION TITLE
Disable editing of Items in Many-To-Many relation ships.

### DISCUSSION THREAD

**Discussion by CodeAndWeb** [directus/directus#5727] [Feature Requests]
I'd like to be able to set the value on the Junction table but not change the selected item from the related table.
I am currently able to edit the related table.

<img width="837" alt="2021-05-17_18-05-03" src="https://user-images.githubusercontent.com/716009/118520752-9d4d3100-b73a-11eb-815f-f61d319467e4.png">

**Comment by benhaynes** [comment]
This goes back to a conversation @rijkvanzanten and I were having about the difference between "normal" permissions, and "relational" permissions. In other words, allowing permissions to be set based on _where_ you're editing the item... on the parent, or via another item.

**Reply by rijkvanzanten** [reply]
https://github.com/directus/directus/discussions/3144 🙂

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
